tgui/packages/tgui/interfaces/XenoartifactConsole.js

```diff
@@ -92,7 +102,7 @@ export const XenoartifactListingBuy = (props, context) => {
           {`${dialogue}`}
         </BlockQuote>
         <Button onClick={() => act(id)}>
-          {`Purchase: ${price} credits.`} <Icon name="shopping-cart" />
```

Comment:
Sorry I didn't mention this earlier actually but remove the space as well and move the icon into the button attribute for icons like most buttons are
icon="shopping-cart" will work the same

Reply:
Yep, added it to documentation images & CL

Reply:
Also applied this to export menu.